devops/providers/azure-devops/templates/infrastructure/scripts/set-cicd-flag.yml

```diff
@@ -0,0 +1,59 @@
+parameters:
```

Comment:
Was hoping to find a way to easily set parameters as required. Unfortunately there is no way to do this simply :/

https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops#required-parameters

Reply:
I'm not convinced the current behavior handles this in a consistent way either. Would you be open to creating a separate issue to look into better handling required parameters/variables across our pipelines?

Reply:
https://github.com/microsoft/cobalt/issues/329